lang/en/texts/mobile-app-open-food-facts.html

```diff
@@ -57,4 +59,29 @@ <h1>Install the Open Food Facts mobile app</h1>
 </scripts>
 
 <initjs>
-</initjs>
\ No newline at end of file
+</initjs>
+
+<br />
+
+<div class="panel callout">
+<h2>It's new !</h2>
```

Comment:
```suggestion
<h2>It's new!</h2>
```
Unnecessary space. 
```suggestion
<h2>What's new?</h2>
```
If you want to use this as a new blog (keep in mind that this could continually break translations in Crowdin), the usual "what's new?" could be better.